src/main/java/carracing/CarRacing.java

```diff
@@ -0,0 +1,31 @@
+package carracing;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class CarRacing {
+
+    public void StartRacingGame(){
+
+        InputView inputView = new InputView();
+        ResultView resultView = new ResultView();
+        List<Car> cars = new ArrayList<Car>();
+        int carsValue = inputView.getInputValue("자동차 대수는 몇 대 인가요?");
```

Comment:
```suggestion
        int carsValue = inputView.requestCarCount();
```

예를 들어 이런식으로 무슨 메세지를 출력할지에 대한 책임을 출력 계층이 담당하도록 하면 어떨까요?

Reply:
mvc패턴 을 적용해보고 싶은데 PR을 일단 하기 위해, CarRacing 에서 출력 계층이 담당하도록 수정 하였습니다.